test/integration/cli-image-create.test.js

```diff
@@ -260,6 +258,66 @@ test('triton image ...', testOpts, function (suite) {
         }
     });
 
+    suite.test('   triton image update ...', function (t) {
+        var argv = ['image', 'update', img.id,
+            'description=this is a description'];
+        h.safeTriton(t, argv, function (err) {
+            if (err) {
+                t.end();
+                return;
+            }
+            argv = ['image', 'get', '-j', img.id];
+            h.safeTriton(t, argv, function (err2, stdout2) {
+                t.ifErr(err2, 'image get response');
+                if (err2) {
+                    t.end();
+                    return;
+                }
+                var result = JSON.parse(stdout2);
+                t.ok(result, 'image update result');
+                t.comment(result.description);
+                t.ok(result.description, 'image update result.description');
+                if (result.description) {
+                    t.equal(result.description, 'this is a description',
+                        'image update description');
+                } else {
+                    t.fail('image update result does not contain description');
+                }
+                t.end();
+            });
+        });
+    });
+
+    suite.test('   triton image tag', function (t) {
+        var argv = ['image', 'tag', img.id,
+            'foo="bar"', 'bool=true', 'one=1'];
+        h.safeTriton(t, argv, function (err) {
+            if (err) {
+                t.end();
+                return;
+            }
+            argv = ['image', 'get', '-j', img.id];
+            h.safeTriton(t, argv, function (err2, stdout2) {
+                t.ifErr(err2, 'image get response');
+                if (err2) {
+                    t.end();
+                    return;
+                }
+                var result = JSON.parse(stdout2);
+                t.ok(result, 'image tag result');
+                t.ok(result.tags, 'image tag result.tags');
+                if (result.tags) {
+                    t.ok(result.tags.foo, 'result.tags.foo');
+                    t.ok(result.tags.one, 'result.tags.one');
+                    t.ok(result.tags.bool, 'result.tags.bool');
+                } else {
+                    t.fail('image tag result does not contain tags');
+                }
+                t.end();
+            });
+        });
+    });
+
```

Comment:
Maybe add tests for update of an acl using an array, and update of tags using object format (triton image update ...).

Reply:
I've removed the acl update option.